numba/cuda/cudadrv/driver.py

```diff
@@ -1038,14 +1037,8 @@ def __init__(self, context, handle, info_log, finalizer=None):
         self.handle = handle
         self.info_log = info_log
         self.finalizer = finalizer
```

Comment:
I have opened a separate PR (#1860) to cleanup things.  I will address it there.